Vagrantfile

```diff
@@ -0,0 +1,47 @@
+# Defines our Vagrant environment
+#
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+my_machines={
+# 'hostname' => ['IPAddress','Memory in MB','Number of CPUs'],
+  'node1'  => ['10.1.15.10','1024','1']
+}
+
+$setupScript = <<SCRIPT
+echo provisioning docker...
+sudo apt-get update
+sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+sudo apt-add-repository 'deb https://apt.dockerproject.org/repo ubuntu-trusty main'
+sudo apt-get update
+sudo apt-get -o Dpkg::Options::="--force-confnew" install --force-yes -y docker-engine=1.10.2-0~trusty
+sudo usermod -a -G docker vagrant
+curl -L "https://github.com/docker/compose/releases/download/1.6.2/docker-compose-$(uname -s)-$(uname -m)" > /usr/local/bin/docker-compose
```

Comment:
Then we should also consider validating the SHA256 sum in the `install-docker.sh` script.  
https://github.com/devops-coop/ansible-minecraft/blob/master/scripts/install-docker.sh#L5-L8  
  
Alternatively, I'll also look into using pip for this. Brief testing shows the below code has the same intended outcome:
```
$ sudo apt-get install python-pip
$ sudo pip install --upgrade pip
$ sudo pip install docker-compose==1.6.2
$ sudo docker-compose version                                
docker-compose version 1.6.2, build 4d72027
```